tests/end_to_end/test_create_new_template.py

```diff
@@ -157,10 +87,76 @@ def create_new_template(page):
     assert "Test message for e2e test" in page.content()
 
 
-def test_create_new_template(authenticated_page):
-    page = authenticated_page
+def test_create_new_template(end_to_end_context):
+    page = end_to_end_context.new_page()
+    page.goto(f"{E2E_TEST_URI}/sign-in")
+    # Wait for the next page to fully load.
+    page.wait_for_load_state("domcontentloaded")
+
+    current_date_time = datetime.datetime.now()
+    new_service_name = "E2E Federal Test Service {now} - {browser_type}".format(
+        now=current_date_time.strftime("%m/%d/%Y %H:%M:%S"),
+        browser_type=page.context.browser.browser_type.name,
+    )
+    page.goto(f"{E2E_TEST_URI}/accounts")
+
+    # Check to make sure that we've arrived at the next page.
+    page.wait_for_load_state("domcontentloaded")
+
+    # Check to make sure that we've arrived at the next page.
+    # Check the page title exists and matches what we expect.
+    expect(page).to_have_title(re.compile("Choose service"))
+
+    # Check for the sign in heading.
+    sign_in_heading = page.get_by_role("heading", name="Choose service")
+    expect(sign_in_heading).to_be_visible()
+
+    # Retrieve some prominent elements on the page for testing.
+    add_service_button = page.get_by_role(
+        "button", name=re.compile("Add a new service")
+    )
+
+    expect(add_service_button).to_be_visible()
+
+    existing_service_link = page.get_by_role("link", name=new_service_name)
 
-    _setup(page)
+    # Check to see if the service was already created - if so, we should fail.
+    # TODO:  Figure out how to make this truly isolated, and/or work in a
+    #        delete service workflow.
+    expect(existing_service_link).to_have_count(0)
```

Comment:
I wonder if there's a way to improve these tests so they're a bit more isolated and the service creation is truly handled separately and only once.  Maybe once we get this fully operationalized we can follow-up on this and reconfigure them somewhat so a "create service" test always runs ahead of most of the others...

I realize that's basically forcing tests to be dependent on one another so they have to run in a specific order, which is not ideal, so there's probably some trade-offs to think about.